### PR TITLE
Fix permission issue for CLEF tasks when on organizer page

### PR DESCRIPTION
**Pull request by ieggel**
@nemaniarjun 
Discovered a bug where ordinary participants were able to see the dataset files on the organizer page, even whey they did not sign an EUA. This has been fixed now.
This PR can be safely merged, only changed permissions in some clef_specific views.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -655,6 +655,36 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "merit_activity_logs", force: :cascade do |t|
```
Comment (piotrekpasciak) [inline db/schema.rb]: This should not be part of this PR. These tables were removed from the repository lately. Could you make sure you've run following migration:
https://github.com/AIcrowd/AIcrowd/blob/master/db/migrate/20200115123518_remove_merit_database_tables.rb

If that doesn't help, can you rebuild your database in development environment to get rid . of these changes?

Comment (piotrekpasciak) [reply db/schema.rb]: These tables are still in schema.rb on your branch. Could you remove them?

Comment (ieggel) [reply db/schema.rb]: I removed them now. Hope everything is OK now.

File: db/schema.rb [hunk 2]
```diff
@@ -647,14 +647,6 @@
     t.integer "participant_id"
   end
 
-  create_table "migration_mappings", force: :cascade do |t|
```
[the remark this answers is not in the document]
Comment (ieggel) [reply db/schema.rb]: My bad. Really sorry for this. Will push in a few seconds.

[the remark this answers is not in the document]
Comment (piotrekpasciak) [reply db/schema.rb]: No need to say sorry. We do Code Review to catch issues like that :)

**Comment by piotrekpasciak** [comment]
@ieggel I've approved it. Feel free to merge the PR whenever you want to.
